src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -90,10 +90,12 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
   }
 
   private var sessionUsed = false
+  private val metricsCategory = s"cassandra.journal.${context.system.name}.${self.path.name}"
 
   private[journal] lazy val cassandraSession: CassandraSession = {
     retry(config.connectionRetries + 1, config.connectionRetryDelay.toMillis) {
       val underlying: Session = clusterBuilder.build().connect()
+      CassandraMetricsRegistry(context.system).addMetrics(metricsCategory, underlying.getCluster.getMetrics.getRegistry)
```

Comment:
sanity check: is it supported for all versions of C* that we support, i.e. 2.1.6 or higher?